docs/inspec_and_friends.md

```diff
@@ -40,6 +40,14 @@ control "sshd-11" do
 end
 ```
 
+That said, InSpec is not RSpec. InSpec does not support arbitrary RSpec features. Some key differences:
+
+ * RSpec `describe` blocks are intended to be nested. In InSpec, `describe` 
+ blocks should not be nested; instead use `control` blocks to describe a 
```

Comment:
A code sample could help with the comparison story that you are describing.

Reply:
The language construct `control` is mention here. Would adding a link to the appropriate to an appropriate docs page?